apple/testing/default_runner/ios_xctestrun_runner.template.sh

```diff
@@ -181,18 +181,16 @@ xctestrun_only_test_section=""
 # Any test prefixed with '-' will be passed to 'SkipTestIdentifiers'. Otherwise
 # the tests is passed to 'OnlyTestIdentifiers',
 if [[ -n "${TESTBRIDGE_TEST_ONLY:-}" || -n "${TEST_FILTER:-}" ]]; then
-  saved_IFS=$IFS
-  IFS=","
-
   if [[ -n "${TESTBRIDGE_TEST_ONLY:-}" && -n "${TEST_FILTER:-}" ]]; then
-    ALL_TESTS=("$TESTBRIDGE_TEST_ONLY,$TEST_FILTER")
+    ALL_TESTS="$TESTBRIDGE_TEST_ONLY,$TEST_FILTER"
   elif [[ -n "${TESTBRIDGE_TEST_ONLY:-}" ]]; then
-    ALL_TESTS=("$TESTBRIDGE_TEST_ONLY")
+    ALL_TESTS="$TESTBRIDGE_TEST_ONLY"
   else
-    ALL_TESTS=("$TEST_FILTER")
+    ALL_TESTS="$TEST_FILTER"
   fi
 
-  for TEST in $ALL_TESTS; do
+  saved_IFS=$IFS
+  IFS=","; for TEST in $ALL_TESTS; do
```

Comment:
this is just stylistic to make it more clear that this is the only place we care about this value being set